File: day-46-morning-exercise/math.js
function summer(argOne, argTwo){
    return argOne + argTwo
}

function multiplier(argOne, argTwo){
    return argOne * argTwo
}

function combiner(argOne, argTwo){
    let argSum = summer(argOne, argTwo)
    let argProduct = multiplier(argOne, argTwo)
    let argAnswer = []
    argAnswer.push(argSum, argProduct)
    return argAnswer
}

let argAnswer = []

argAnswer = combiner(5, 10)

console.log(argAnswer)

File: day-46-morning-exercise/math.rb
def summer(arg_one, arg_two)
  arg_one + arg_two
end

def multiplier(arg_one, arg_two)
  arg_one * arg_two
end

def combiner(arg_one, arg_two)
  arg_sum = summer(arg_one, arg_two)
  arg_product = multiplier(arg_one, arg_two)
  arg_answer = []
  arg_answer.push(arg_sum, arg_product)
end

arg_answer = combiner(5, 10)

p arg_answer
